make/migrations/postgresql/0010_1.9.0_schema.up.sql

```diff
@@ -0,0 +1,10 @@
+/* add table for CVE whitelist */
```

Comment:
What is wrong with filename `0005_1.9.0_schema.up.sql` ?

Reply:
Just leave some buffer for 1.8.x patch release.